cmd/ringbuf/src/lib.rs

```diff
@@ -151,16 +151,32 @@ fn ringbuf(
     let mut ringbufs = vec![];
 
     for v in hubris.qualified_variables() {
+        let def = match hubris.lookup_struct(v.1.goff) {
+            Ok(s) => {
+                // Skip variables whose type does not indicate they contain a
+                // ringbuf; this check is imprecise but probably good enough
+                if s.name.contains("Ringbuf") {
```

Comment:
FWIW my first shot here was `"ringbuf::Ringbuf"`, which worked on the latest archive I'm using from hubris but failed for 99 of the committed tests. My second try was `Ringbuf<`, which failed for only two tests (which I find somewhat surprising). `Ringbuf` passes all tests and is probably fine, but if you have insight into why the other two failed I'd appreciate hearing it.